secretmanager/snippets/regional_samples/update_regional_secret_with_delayed_destroy.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
+
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+
+import argparse
+
+# [START secretmanager_update_regional_secret_delayed_destroy]
+
+# Import the Secret Manager client library.
+from google.cloud import secretmanager_v1
+from google.protobuf.duration_pb2 import Duration
+
+
+def update_regional_secret_with_delayed_destroy(
+    project_id: str, location_id: str, secret_id: str, new_version_destroy_ttl: int
+) -> secretmanager_v1.UpdateSecretRequest:
+    """
+    Update the version destroy ttl on an existing secret.
```

Comment:
suggestion: please consider adding argument and return value explanation to the docstring